specification/web/resource-manager/Microsoft.Web/stable/2018-02-01/AppServiceEnvironments.json

```diff
@@ -2177,6 +2306,30 @@
         }
       }
     },
+    "InboundEnvironmentEndpoint": {
+      "description": "Endpoints for a particular type",
+      "type": "object",
+      "properties": {
+        "description": {
+          "description": "Text describing the endpoints.",
```

Comment:
Please add a description

Reply:
This is a short description of what the endpoints is accessed for. E.G. "App Service management", "App Service Environment VIP", or "App Service Environment subnet".
Does this more accurately represent this? 
"Short description of why the endpoints are accessed."